src/main/java/com/synopsys/integration/alert/channel/jira/JiraIssueHandler.java

```diff
@@ -150,8 +151,11 @@ public String createOrUpdateIssues(final FieldAccessor fieldAccessor, final Mess
                     if (ItemOperation.ADD.equals(operation) || ItemOperation.UPDATE.equals(operation)) {
                         fieldsBuilder.setProject(projectId);
                         fieldsBuilder.setIssueType(issueType);
-                        final String username = fieldAccessor.getString(JiraDescriptor.KEY_JIRA_ADMIN_EMAIL_ADDRESS).orElseThrow(() -> new AlertException("Expected to be passed a jira user email address."));
-                        final IssueResponseModel issue = issueService.createIssue(new IssueCreationRequestModel(username, issueType, projectName, fieldsBuilder, List.of()));
+                        String issueCreator = fieldAccessor.getString(JiraDescriptor.KEY_ISSUE_CREATOR)
+                                                  .filter(StringUtils::isNotBlank)
+                                                  .or(() -> fieldAccessor.getString(JiraDescriptor.KEY_JIRA_ADMIN_EMAIL_ADDRESS))
+                                                  .orElseThrow(() -> new AlertException("Expected to be passed a jira user email address."));
```

Comment:
AlertException could be an AlertFieldException. That would make the error easier to understand in the UI.